src/models/Message.js

```diff
@@ -0,0 +1,50 @@
+import User from "./User";
+
+class Message {
+  static mimicRealmMappedPropertiesSchema( msg ) {
+
+    const fromUser = User.mapApiToRealm( msg.from_user );
+    const toUser = User.mapApiToRealm( msg.to_user );
+
+    return {
+      ...msg,
+      subject: msg.subject,
+      body: msg.body,
+      fromUser,
+      toUser,
+      threadId: msg.thread_id
+    };
+  }
+
+  static createMessageForRealm( msg, realm ) {
+    const newMsg = {
+      ...msg,
+        subject: msg.subject,
+        body: msg.body,
+        fromUser: User.mapApiToRealm( msg.from_user, realm ),
```

Comment:
These should be snake case, right?

Reply:
@albullington was mimic'ing `class Observation`.  A little unsure when we use snake vs camel